src/lib.rs

```diff
@@ -107,6 +107,7 @@ impl ParseOptions {
 pub struct DemangleOptions {
     no_params: bool,
     no_return_type: bool,
+    simplify_template_parameters: bool,
```

Comment:
I think this is too vague and we should just pass through the `show_expression_literal_types` option verbatim.